--- wop-0.4.3/src/scusibot.cpp	2005-10-16 19:53:05.000000000 +0200
+++ wop-0.4.3.new/src/scusibot.cpp	2008-03-20 10:52:07.000000000 +0100
@@ -5,6 +5,7 @@
 #include "stationarygun.hpp"
 #include "global.hpp"
 #include <vector>
+#include <algorithm>
 
 ScusiBot::ScusiBot()
 	: m_event( Event::EMPTY ), m_count( 0 ), m_state( IDLE ) {
--- wop-0.4.3/src/gameinfowindow.cpp	2006-03-27 12:27:52.000000000 +0200
+++ wop-0.4.3.new/src/gameinfowindow.cpp	2009-04-01 10:10:11.000000000 +0200
@@ -10,6 +10,8 @@
 #include "avatarworm.hpp"
 #include "wopsprites.hpp"
 #include "wopsprites.cpp"
+#include "spriteset.cpp"
+#include "spritesequence.cpp"
 #include "scorekeeper.hpp"
 #include "wopgui.hpp"
 
--- wop-0.4.3/src/map.cpp	2006-01-20 12:23:30.000000000 +0100
+++ wop-0.4.3.new/src/map.cpp	2009-04-01 10:08:41.000000000 +0200
@@ -9,6 +9,8 @@
 #include "world.hpp"
 #include "wopsettings.hpp"
 #include "objectpaths.hpp"
+#include "spriteset.cpp"
+#include "spritesequence.cpp"
 
 /**********************************************************/
 
